### ISSUE TITLE
Image not scaling responsively both width & height

### ISSUE THREAD

**Issue by eoghanmccarthy** [DominicTobias/react-image-crop#174]
Hello

ReactCrop container doesn't seem to be containing the img (ReactCrop__image) when scaling. I can have it scale at smaller sizes but when the ReactCrop container height reaches my component height ReactCrop__image continues to grow.

Is there any CSS examples where the container, image and crop selection all scale proportionally and when boundaries are met all 3 stop scaling? My latest attempt is below where ReactCrop div is positioned centre centre in my component. 

```
.component {
	position: relative;
	width: 100%;
	height: calc(100vh - 480px);
	.ReactCrop {
		display: block;
		position: absolute;
		top: 50%;
		left: 50%;
		width: auto;
		height: auto;
		max-height: 100%;
		transform: translate(-50%, -50%);
		.ReactCrop__image {
			display: block;
			width: auto;
			max-width: 100%;
			height: auto;
			max-height: 100%;
		}
	}
}
```

Cheers

**Comment by eoghanmccarthy** [comment]
I got it. Good to know if anyone else is interested. Change ReactCrop__image to: 

```
.ReactCrop__image {
display: block;
max-width: 100%;
max-height: fill-available;
		}
```

**Comment by DominicTobias** [comment]
Released in v4.0.0 let me know if you have any problems, as mentioned this is only a breaking change for people who had overridden `$mobile-media-query` scss variable

**Comment by hamczu** [comment]
I have an issue with that change. CSS autoprefixer breaks the build with the error:
"Replace fill-available to stretch, because spec had been changed"

I believe the reason for that and potential fix is well described here:
https://github.com/postcss/autoprefixer/issues/1035

**Comment by DominicTobias** [comment]
Thanks @hamczu will publish a patch fix shortly

**Comment by DominicTobias** [comment]
Thanks @hamczu this has been published now at 4.0.1, please let me know if that fixes your issue

**Comment by hamczu** [comment]
@DominicTobias fixed! thx a lot!

**Comment by jedierikb** [comment]
It seems this fix is no longer in v8.5
Adding it fixed issue as described.
Is there a best practice for adding this css flag?

**Comment by jedierikb** [comment]
see here for fix: https://github.com/DominicTobias/react-image-crop/releases/tag/8.3.0
